src/screens/Settings/SettingsList/index.js

```diff
@@ -150,17 +151,21 @@ class SettingsListScreen extends NavigatorComponent<Props, State> {
               />
             </View>
           </View>
+          <TouchableOpacity
+            style={styles.itemViewStyle}
+            onPress={() => { this.props.navigator.push(screen('CONFIRM_KEY_INSTRUCTION_SCREEN')); }
+        }
+          >
+            <Text style={styles.itemSelectText}>
+              {i18n.t('screens.settings.confirmPrivateKey')}
+            </Text>
+
+          </TouchableOpacity>
           <View style={styles.buttonViewStyle}>
             <Button
               rounded
-              onPress={() =>
-                i18next.changeLanguage('hi', (err, t) => {
-                  if (err) {
-                    return console.log('something went wrong loading', err);
-                  }
-                  t('hi'); // -> same as i18next.t
```

Comment:
@hunght in case anyone was affected by this bug, they will already have their language  setting stuck on 'hi' and they will still be unable to switch back to 'en'. Can you simply change this to the way it was before but instead of 'hi' use 'en'. That way, anyone who was affected will get the 'en' back. When we actually have support for the other languages we can remove that code.